protos/deployment.proto

```diff
@@ -182,7 +182,7 @@ message ListDeploymentsRequest {
   DeploymentSpec.DeploymentOperator operator = 4;
   SORTABLE_COLUMN order_by = 5;
   bool ascending_order = 6;
-  map<string, string> labels = 7;
+  string labels_query =7;
```

Comment:
add a comment here about the query syntax, and a note about the `in`, `notin` query syntax that we plan to add in the future?